frontend/package.json

```diff
@@ -16,7 +16,7 @@
     "reactstrap": "^7.1.0"
   },
   "devDependencies": {
-    "eslint": "5.12.0",
+    "eslint": "^5.12.0",
```

Comment:
What's the reason for the update to the `^`? I only ask because I've been bitten in the past by adding the `^` or `~`, and having a package update break my code.

Reply:
I'm not sure... I didn't actually make that change. But that's a good point.